cmd/notary/delegations.go

```diff
@@ -38,8 +38,8 @@ var cmdDelegationAddTemplate = usageTemplate{
 	Long:  "Add a keys to delegation using the provided public key PEM encoded X509 certificates in a specific Global Unique Name.",
 }
 
-var paths []string
-var removeAll, removeYes bool
+var addPaths, removePaths []string
+var allPaths, removeAll, removeYes bool
```

Comment:
We probably want to use the `-y` flag in other places (or will do), so I'd just call it `forceYes` or something more generic than `removeYes`